src/components/overview/VariantsSummary.js

```diff
@@ -4,17 +4,24 @@ import { Col, Row, Spin, Statistic, Typography, Icon } from "antd";
 
 const VariantsSummary = ( ) => {
 
-    const fetchingTableSummaries = useSelector(state => state.tableSummaries?.isFetching);
-    const numVCFs =
-        useSelector((state) => state.overviewSummary.data?.data_type_specific?.experiment_results.file_format.VCF) || 0;
+    const fetchingVariantsOverview = useSelector(state => state.explorer?.fetchingVariantsOverview);
+    const variantsOverviewResults = useSelector((state) => state.explorer.variantsOverviewResponse);
```

Comment:
not sure here why one line has optional chaining and the other doesn't...

Reply:
neither need optional chaining im pretty sure, explorer should always at the very least be `{}`

Reply:
Patched the edge case for when gohan request fails